go/common/grpc/cbor.go

```diff
@@ -1,9 +1,14 @@
 package grpc
 
 import (
+	"google.golang.org/grpc/encoding"
+
 	"github.com/oasislabs/oasis-core/go/common/cbor"
 )
 
+// CBORCodecName is the name of the CBOR gRPC codec.
+const CBORCodecName = "cbor"
```

Comment:
There is none, because the Rust side doesn't have a server part and our gRPC service now forces the use of CBOR so subtypes are ignored anyway. This was used in the interim while the Go part was being ported.

Since it is unused, we could make it private again.

Reply:
but wouldn't we be better citizens if we were to send that subtype anyway?